autoload/neomake/utils.vim

```diff
@@ -181,6 +181,9 @@ function! neomake#utils#MakerIsAvailable(ft, maker_name) abort
     endif
     if !has_key(s:available_makers, a:maker_name)
         let maker = neomake#GetMaker(a:maker_name, a:ft)
+        if maker == {}
```

Comment:
`maker == {}` or `empty(maker)`? :-)

I have no real favourite, but I guess we should decide for one and be consistent with all new changes.